packages/ffe-grid/less/ffe-grid.less

```diff
@@ -15,6 +20,36 @@
     .create-column-offset(@size, @total, @current + 1);
 }
 
+.ffe-grid {
+    margin: 0 auto;
+    padding: 0 @ffe-spacing-sm;
```

Comment:
mye mulig jag misførstår noe nå men kan man ha background på en row hvis griden har padding?
Bare for og prova illustrera vad jag mener. 
![image](https://user-images.githubusercontent.com/2248579/195616073-b7e0e1d1-2120-4116-a668-5006c4dbeb34.png)

Reply:
Godt poeng, paddingen må selvfølgelig legges inn på row og ikke på containeren. Jeg fikser det, modifierne forblir de samme.